lib/jxl/epf.cc

```diff
@@ -55,19 +55,26 @@ using DU = HWY_CAPPED(uint32_t, GroupBorderAssigner::kPaddingXRound);
 // kInvSigmaNum / 0.3
 constexpr float kMinSigma = -3.90524291751269967465540850526868f;
 
+namespace {
+constexpr float kSlope = 64.0f;
```

Comment:
Why 64?
I suppose a power of two has advantages, but other than that, why 2^6 and not 2^5 or 2^7?

Reply:
128 is too much (it ~doubles the error), 64 is the highest power of two that gets the same error as no EPF (and thus the one that changes the results the least)